common/changes/office-ui-fabric-react/kathayer-portcommandbarfixes_2018-05-03-17-07.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Port CommandBar accessibility fixes to 6.0",
```

Comment:
In this comment, it would be good to reference the origin of the port (e.g. a pr number (`#1234`) or GitHub commit SHA)